packages/lit-dev-tools-esm/package.json

```
@@ -31,13 +32,24 @@
       ],
       "command": "../../node_modules/.bin/tsc --build --pretty",
       "clean": "if-file-deleted"
+    },
+    "publish-search-index": {
+      "dependencies": [
+        "build"
+      ],
```

Need to build content first so it generates `searchIndex.json`.
```suggestion
      "dependencies": [
        "../lit-dev-content:build",
        "build"
      ],
```